### PR TITLE
Update to Twirl 1.3.0

### PR DESCRIPTION
**Pull request by danielnixon**

### CODE REVIEW

File: src/main/twirl/scalajs/jsdeps.scala.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import play.twirl.api.TwirlFeatureImports.defining
```
Comment (vmunier) [inline src/main/twirl/scalajs/jsdeps.scala.html]: The project compiles fine without the import. What is the reason for adding this import?

**Comment by vmunier** [comment]
Hi @danielnixon, just one comment about the imports, which do not seem to be needed.

File: src/main/twirl/scalajs/jsdeps.scala.html (hunk 1 above)
Comment (danielnixon) [inline src/main/twirl/scalajs/jsdeps.scala.html]: `play.twirl.api.TemplateMagic` (where `defining` is defined) is deprecated. Importing `TwirlFeatureImports` avoids compiler warnings and allows scalajs-scripts users to turn on the `-Xfatal-warnings` scalac option.

Comment (danielnixon) [inline src/main/twirl/scalajs/jsdeps.scala.html]: See https://github.com/playframework/twirl/commit/cca9cd8c450205bb5b1fed33b5614bd46dccaa6f

Comment (vmunier) [inline src/main/twirl/scalajs/jsdeps.scala.html]: I do not see any warnings when compiling the code. What compiler options & Scala version are you using?

Comment (danielnixon) [inline src/main/twirl/scalajs/jsdeps.scala.html]: My bad. That was an issue with Twirl 1.2.1 but apparently not with 1.3.0.

Comment (vmunier) [inline src/main/twirl/scalajs/jsdeps.scala.html]: No worries, I've just pushed a commit to upgrade to several libraries (including Twirl `v1.3.0`) https://github.com/vmunier/scalajs-scripts/commit/fab19c75f4b93242354ac87b7a44c7d0086870c0

**Comment by vmunier** [comment]
This [commit](https://github.com/vmunier/scalajs-scripts/commit/fab19c75f4b93242354ac87b7a44c7d0086870c0) includes the upgrade to Twirl `v1.3.0`.

**Comment by danielnixon** [comment]
Thanks @vmunier !

**Pull request closed without merging**